src/components/VaultParamChange.tsx

```diff
@@ -0,0 +1,219 @@
+
+  console.log('ProposeParamChange', { data, paramPatch });
+
+  const canGovern = !!props.charterOfferId;
+
+  function displayParam(name: string, { type, value }: ParameterValue) {
+    console.log('display param', name, value, type);
+    switch (type) {
+      case 'amount':
+        return (
+          <AmountInput
+            suffix={name === 'DebtLimit' && 'IST'}
+            value={(paramPatch[name] || value).value}
+            brand={value.brand}
+            onChange={newVal =>
+              setParamPatch({
+                ...paramPatch,
+                [name]: { brand: value.brand, value: newVal },
+              })
+            }
+          />
+        );
+      case 'ratio':
+        return (
+          <PercentageInput
+            ratio={paramPatch[name] || value}
+            onChange={newRatio =>
+              setParamPatch({
+                ...paramPatch,
+                [name]: newRatio,
+              })
+            }
+          />
+        );
+      default:
+        return <i className="text-gray-500">{type} not supported</i>;
+    }
+  }
+
+  function handleSubmit(event) {
+    event.preventDefault();
+    console.log({ event });
+    const offer = walletUtils.makeVoteOnVaultManagerParams(
+      props.charterOfferId,
+      collateralBrand,
+      paramPatch,
+      minutesUntilClose,
+    );
+    void walletUtils.sendOffer(offer);
+  }
+
+  let content = (
+    <div className="text-gray-500 mt-2">
+      <i>Waiting for existing parameter values...</i>
+    </div>
+  );
+
+  if (status === LoadStatus.Received) {
+    content = (
+      <motion.div
+        className="px-1 overflow-hidden"
+        initial={{ height: 0, opacity: 0 }}
+        animate={{ height: 'auto', opacity: 1 }}
+        transition={{ type: 'tween' }}
+      >
+        <form onSubmit={handleSubmit}>
+          {Object.entries(data.current).map(([name, value]) => (
+            <div className="mb-2" key={name}>
+              <label className="block">
+                <span className="text-gray-700">{name}</span>
+                <div className="w-full">
+                  {displayParam(name, value as ParameterValue)}
+                </div>
+              </label>
+            </div>
+          ))}
+          <label className="block">
+            <span className="text-gray-700">Minutes until close of vote</span>
+            <input
+              type="number"
+              className="rounded mt-1 block w-full border-gray-300 focus:border-purple-300 focus:ring-purple-300"
+              value={minutesUntilClose}
+              onChange={e => setMinutesUntilClose(e.target.valueAsNumber)}
+            />
+          </label>
+          <div className="w-full flex flex-row justify-end mt-2">
+            <input
+              type="submit"
+              value="Propose Parameter Change"
+              className={clsx(
+                'btn-primary p-2 rounded mt-2',
+                canGovern ? 'cursor-pointer' : 'cursor-not-allowed',
```

Comment:
TIL cursor-not-allowed

feels like overkill that we might not remember to do everywhere, but doesn't hurt